src/Avalonia.Controls.ColorPicker/Themes/Fluent/ColorView.xaml

```diff
@@ -0,0 +1,562 @@
+                  VerticalAlignment="Stretch"
+                  Background="{TemplateBinding Background}" />
+          <Border Grid.Row="0"
+                  Grid.RowSpan="2"
+                  Margin="0,48,0,0"
+                  HorizontalAlignment="Stretch"
+                  VerticalAlignment="Stretch"
+                  Background="{ThemeResource LayerFillColorAltBrush}"
+                  BorderBrush="{ThemeResource DividerStrokeColorDefaultBrush}"
+                  BorderThickness="0,1,0,0" />-->
+
+          <TabControl Grid.Row="0"
+                      Height="338"
+                      Width="350">
+            <TabControl.Resources>
+              <Thickness x:Key="TabItemMargin">0,0,0,0</Thickness>
+            </TabControl.Resources>
+            <TabControl.Styles>
+              <Style Selector="TabItem">
+                <Setter Property="Padding" Value="0" />
+              </Style>
+            </TabControl.Styles>
+            <!-- Spectrum Tab -->
+            <TabItem>
+              <TabItem.Header>
+                <Border Width="116"
+                        Height="{DynamicResource ColorViewTabStripHeight}">
+                  <PathIcon Width="20"
+                            Height="20"
+                            Data="{DynamicResource ColorViewSpectrumIconGeometry}" />
+                </Border>
+              </TabItem.Header>
+              <Grid ColumnDefinitions="Auto,*,Auto"
+                    RowDefinitions="Auto,*"
+                    Margin="12">
+                <primitives:ColorSlider x:Name="ColorSpectrumThirdComponentSlider"
+                                        AutomationProperties.Name="Third Component"
+                                        Grid.Column="0"
+                                        Grid.Row="1"
+                                        Grid.RowSpan="2"
+                                        IsAlphaMaxForced="True"
+                                        IsSaturationValueMaxForced="False"
+                                        Orientation="Vertical"
+                                        ColorModel="Hsva"
+                                        ColorComponent="{Binding Components, ElementName=ColorSpectrum, Converter={StaticResource ThirdComponent}}"
+                                        HsvColor="{Binding HsvColor, ElementName=ColorSpectrum}"
+                                        HorizontalAlignment="Center"
+                                        VerticalAlignment="Stretch"
+                                        Margin="0,0,12,0" />
+                <primitives:ColorSpectrum x:Name="ColorSpectrum"
+                                          Grid.Column="1"
+                                          Grid.Row="0"
+                                          Grid.RowSpan="2"
+                                          Components="{TemplateBinding ColorSpectrumComponents}"
+                                          HsvColor="{Binding $parent[ColorView].HsvColor}"
```

Comment:
@maxkatz6 I have a question about binding here. `{TemplateBinding HsvColor}` doesn't seem to work in this case so I've had to replace it with `{Binding $parent[ColorView].HsvColor}`. This applies several places. Do you know why that might be? This really should be fixed so normal TemplateBinding can be used.

Reply:
This is because TemplateBinding is only possible OneWay (I haven't thought about that in a while, I guess it just rarely comes up for me). I'll just use a TemplatedParent relative source then. The assumption is all of this works the same as WPF.

Reply:
I thought you can have `Components="{TemplateBinding ColorSpectrumComponents, Mode=TwoWay}"` but iirc you need to explicitly set it.

Reply:
I had it specified as TwoWay at one point. I can test again. However, if this works like WPF it isnt possible in a TemplateBinding to have TwoWay mode.

Reply:
Avalonia is better than WPF here imo.

Reply:
Well, I do think I tested that specifically. Could have made a mistake though. I was changing a few things at once to figure our what the issue was.

Reply:
@timunie I tested this again and confirmed it doesn't work. It behaves the same as WPF. You have to use the full binding syntax for TwoWay.